wbooks/src/app/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import logoWolox from './assets/LogoWolox.png';
+import './application.css';
+
+class App extends Component {
+  state = {
+    name: 'Nombre',
+    lastname: 'Apellido',
+    email: 'example@domain.com',
+    password: 'password',
+    password_confirm: 'password'
+  };
+
+  render () {
+    return (
+      <React.Fragment>
```

Comment:
You can export directly the Fragment:

```import React, { Component, Fragment } from 'react'```

Reply:
And in this case, The Fragment is not necessary

Reply:
removed it